code-challenges.test.js

```diff
@@ -22,11 +33,52 @@ const fibLength2 = 10
 
 // b) Create the function that makes the test pass.
 
+/* Psuedo Code
+   1. Create a new const with the name fibSequence
+   2. pass (number) into fibSequence
+   3.create a new empty array named fib1
+   4. create two new variables num1 and num2, give them each a value of 1
+   5. create an iteration
+   6. create a variable for num3 which is the sum of num1 + num2
+   7. push the iteration into the empty array (fib1)
+   8. at the end of each iteration, change the value of num1 to become num 2 && 
+      num2 to become num3
+   9. return the name of the created array
+*/
+
+
+
+const fibSequence = (number) => {
+  let fib1 = []
+  let num1 = 1
+  let num2 = 1
+  
+  for (let i = 0; i < number; i++) {
+    
+    let num3 = num1 + num2
+    fib1.push(num1)
+    num1 = num2 
+    num2 = num3
+  }
+  return fib1
+}
+
+
+
 // --------------------2) Create a function that takes in an object and returns an array of the numbers sorted from least to greatest.
 // Hint: Check out this resource: Object.values() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("totalStudyTime", () => {
```

Comment:
Your tests are looking excellent.  To step them up one more notch, I would recommend moving your test variables into your test just after the it statement but before the expect statement.  This will keep the variables confined to only be used within the test.